opsdroid/web.py

```diff
@@ -198,6 +278,15 @@ async def web_index_handler(self, request):
         """
         return self.build_response(200, {"message": "Welcome to the opsdroid API"})
 
+    async def check_request(self, request):
+        client_token = request.headers.get("Authorization")
+        if (
+            client_token is None
+            or client_token != f"Basic {self.authorization_token}"
```

Comment:
If we want to stick to the Basic Auth standard then it needs a `username:password` base64 encoded. Otherwise I'd just remove the `Basic ` not to confuse the user, that we are using Basic Auth.